LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2022- Slack Technologies, LLC
```

Comment:
I don't think we should do this. The license, and code, for this repository existed in 2022 as well.

Reply:
I think we just pulled this over from existing Bolt/SDK docs, if I remember correctly. Whatever the move is we should do it everywhere™️ (I know you're sick of hearing me say that)

Reply:
Yes just the date change 😄

Reply:
Yea, I agree @filmaj the change format it super weird. I'd expect it to say:

```
Copyright (c) 2022-2023 Slack Technologies, LLC
```

I chose this format because [looking at the other audit PRs, it's what I also saw](https://github.com/slack-samples/deno-welcome-bot/pull/22/files#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7).

Happy to change it, but I leaned on consistency